server/src/tests/routes/user.test.ts

```diff
@@ -58,71 +60,35 @@ describe('User routes', () => {
   });
 
   describe('Edit user', () => {
-    test('Edit user', async () => {
```

Comment:
we can comment this for now, but this is working via frontend. just tested, so i think the issue might be in the test case just a heads up for when we try this on drizzle branch

Reply:
As I have commented, the edit user service doesn't return updated data, but as you have tested using the frontend, So it would be updating data, but the function is returning old data, and the function might be doing a refresh to get updated data.

I will still consider it while doing it with drizzle routes.